assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -38,4 +38,22 @@ body {
   .spaced-left {
     margin-left: 0.3em;
   }
-}
\ No newline at end of file
+}
+
+#contact-page {   
+
+  .ui.container {
+    padding: 15px
+  }
+
+  .content-tab {
+    margin-bottom: 10px;
```

Comment:
you could use `mb-10` instead

Reply:
This class is not used anymore, it was a leftover from the migration.